src/LdswScraper/Program.cs

```diff
@@ -0,0 +1,56 @@
+using System.Collections.Concurrent;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using LdswScraper;
+using Microsoft.Extensions.DependencyInjection;
+
+var parallelOption = new Option<int>(
+    new[] { "--parallelism", "-p" },
+    () => 32,
+    "The number of concurrent host fetchers.");
+
+var timeoutOption = new Option<double>(
+    new[] { "--timeout", "-t" },
+    () => 0.5,
+    "The delay between requests to the same host in seconds.");
+
+var rootCommand = new RootCommand("LDSW Scraper")
+{
+    parallelOption,
+    timeoutOption
+};
+
+rootCommand.SetHandler(async (int parallelism, double timeout) =>
+{
+    var config = new ScraperConfig(parallelism, timeout);
+    Console.WriteLine($"Starting scraper with P={parallelism}, T={timeout}s");
+
+    var services = new ServiceCollection();
+    services.AddHttpClient();
+    var serviceProvider = services.BuildServiceProvider();
+    var httpClientFactory = serviceProvider.GetRequiredService<IHttpClientFactory>();
```

Comment:
@jules  Initialize ILogger with console output instead of using Console.WriteLine.